src/test/java/tatracker/logic/parser/module/DeleteModuleCommandParserTest.java

```diff
@@ -0,0 +1,32 @@
+package tatracker.logic.parser.module;
+
+import static tatracker.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static tatracker.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import tatracker.commons.core.Messages;
+import tatracker.logic.commands.module.DeleteModuleCommand;
+
+public class DeleteModuleCommandParserTest {
+
+    private DeleteModuleCommandParser parser = new DeleteModuleCommandParser();
+
+    @Test
+    public void parse_allFieldsPresent_success() {
```

Comment:
Invalid field?

Reply:
there r no invalids in module and group codes os it all works... main checks are done in the command